cl/assets/static-global/css/override.css

```diff
@@ -1169,3 +1169,18 @@ emphasis {
 }
 
 /* END Profile tabs styles */
+
+/* h-captcha submit button styles */
+ button.h-captcha{
+   color: #ffffff;
+   background-color: #428bca;
+   padding: 10px 16px;
+   font-size: 15px;
+   line-height: 1.33;
+   border-radius: 6px;
+   border: 1px solid transparent;
+ }
+ button.h-captcha:hover{
+   background-color: #3071a9;
+ }
+/* END  h-captcha submit button styles */
```

Comment:
I'm surprised this is needed. What happens without it?

Reply:
Yeah, the thing is that h-captcha overrides the original button completely, so the button has no style:
![Screen Shot 2022-07-27 at 9 52 32](https://user-images.githubusercontent.com/486004/181279703-f6913bcc-d2bd-4b13-b4ba-4f795caf00e5.png)


If it's ok, I could leave it with no style.

Working on the other changes, thanks!.